src/main/java/blackjack/domain/card/Rank.java

```diff
@@ -0,0 +1,41 @@
+package blackjack.domain.card;
+
+public enum Rank {
```

Comment:
```suggestion
public enum CardRank {
```

클래스명을 그 자체로 무슨 뜻인지 명확히 알 수 있게 지어보면 어떨까요?

물론 이 미션이 블랙잭 게임을 구현하는 미션이고, Rank 클래스가 `card` 패키지 안에 있으니 이 Rank 클래스는 Card의 Rank임을 '유추'할 수는 있어요. 그러나 **이러한 배경 정보를 바탕으로 '유추'하는 것과 클래스명 그 자체로 알 수 있도록 하는 것은 큰 차이가 있어요.**

프로젝트를 유지보수하면서 클래스는 계속 늘어나고, Rank가 사실은 Card의 Rank였다는 사실을 계속 인지하는 것은 점차 어려워질거에요. (어쩌면 이름은 같고 역할은 다른 Rank 클래스도 생길 수도 있고요.)

앞으로 벌어나지도 않을 일 때문에 미리 그것까지 고려해야될까라는 물음에는 저는 이정도면 충분히 고려를 해야한다고 봐요. 이러한 점들을 종합적으로 고려했을 때, 클래스를 설계하는 시점부터 클래스명을 더욱 명확하게, 의미를 좁히는 방향으로 지어보면 어떨까 싶습니다 :D